test/node/multipart.js

```diff
@@ -148,12 +148,14 @@ describe('Request', function(){
     it('should fire progress event', function(done){
       var loaded = 0;
       var total = 0;
+      var direction = '';
       request
       .post(':3005/echo')
       .attach('document', 'test/node/fixtures/user.html')
       .on('progress', function (event) {
         total = event.total;
         loaded = event.loaded;
+        direction = event.direction;
```

Comment:
if that endpoint echoes, then the file is downloaded as well. It's a bit worrying that there's no download event after the upload.